### ISSUE TITLE
[Bug] Sometimes CUDA check failed

### ISSUE THREAD

**Issue by OneMoreSecond** [tensorflow/tensorflow#22105]
### System information
- **Code**: a keras example (use tf.keras instead) https://github.com/keras-team/keras/blob/master/examples/lstm_seq2seq.py
- **OS Platform and Distribution**: Windows server 2016
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 'v1.10.0-rc1-19-g656e7a2b34' 1.10.0
- **Python version**: 3.6.6
- **CUDA/cuDNN version**: CUDA 9.0, cuDNN 7.0

- **Have I written custom code**: some I/O code to adapt data format, which shouldn't involved
- **Bazel version** : N/A
- **GPU model and memory**: Quadro P600, 2GiB
- **Exact command to reproduce**: python lstm_seq2seq.py
- **Mobile device**: N/A

### Describe the problem
I try to run the code mentioned above.
If I run the script successfully, the script will raise a error in next several runs.
Keep retry and I can successfully run it again.

I only know the check failed, but can't see the error code.
Could you provide some help on it?

Another problem: only changing latent_dim from 128 to 256 results in 5e-8 training loss and a bad model. But CPU training gives a good result.

### Source code / logs
Epoch 1/100
2018-09-06 10:51:48.513539: I T:\src\github\tensorflow\tensorflow\core\platform\cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
2018-09-06 10:51:48.726860: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:1405] Found device 0 with properties:
name: Quadro P600 major: 6 minor: 1 memoryClockRate(GHz): 1.5565
pciBusID: 0000:21:00.0
totalMemory: 2.00GiB freeMemory: 1.62GiB
2018-09-06 10:51:48.731423: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:1484] Adding visible gpu devices: 0
2018-09-06 10:51:49.568323: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
2018-09-06 10:51:49.573347: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:971]      0
2018-09-06 10:51:49.576631: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:984] 0:   N
2018-09-06 10:51:49.582844: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 1377 MB memory) -> physical GPU (device: 0, name:
Quadro P600, pci bus id: 0000:21:00.0, compute capability: 6.1)
2018-09-06 10:51:49.872995: I T:\src\github\tensorflow\tensorflow\core\kernels\cuda_solvers.cc:159] Creating CudaSolver handles for stream 000001DDA2412050
**2018-09-06 10:51:50.120523: F T:\src\github\tensorflow\tensorflow\core\kernels\cuda_solvers.cc:94] Check failed: cusolverDnCreate(&cusolver_dn_handle) == CUSOLVER_STATUS_SUCCESS Failed to create cuSolverDN instance.**

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
Bazel version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by tensorflowbutler** [comment]
It has been 19 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by harshini-gadige** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by ngxson** [comment]
I got the same problem. It's just because I've ran out of GPU memory. Decreasing model parameters and it works again.

**Comment by bionicles** [comment]
same problem. guess it's memory because it's architecture search ... but it would be cool for this to fail more gracefully

**Comment by JingyuQian** [comment]
My colleague had this problem once. Solved by setting `allow_growth=True` when creating session so I'll record it here in case someone need it.

She had 2 GPUs in her machine, GPU 0 was empty and GPU 1 had graphics processes such as Xorg. The model trained well on GPU 0 only but displayed such problem when GPU 1 only.
